### PR TITLE
[Issue #2592] DOCS: deleted duplicate docs in docs/source/examples/Widget Styling.ipynb

### PR DESCRIPTION
**Pull request by jhl0204**
PR for Issue #2592. 

This is my first PR, so please let me know if I'm doing anything wrong! 
I have tested the changes per [Jupyter Guidelines](https://jupyter.readthedocs.io/en/latest/contrib_docs/getting-started.html)

### CODE REVIEW

File: docs/source/examples/Widget Styling.ipynb [hunk 1]
```diff
@@ -1046,19 +1046,6 @@
...
-    "\n",
-    "In addition, some CSS styling is applied to images that is not applied to other widgets: `max_width` is set to `100%` and `height` is set to `auto`. \n",
-    "\n",
-    "You should not rely on `Image.width` or `Image.height` to determine the display width and height of the image widget. Any CSS styling, whether via `Image.layout` or some other source, will override `Image.width` and `Image.height`.\n",
-    "\n",
-    "When displaying an `Image` widget by itself, setting `Image.layout.width` to the desired width will display the `Image` widget with the same aspect ratio as the original image.\n",
-    "\n",
-    "When placing an `Image` inside a `Box` (or `HBox` or `VBox`) the result depends on whether a width has been set on the `Box`. If it has, then the image will be stretched (or compressed) to fit within the box because `Image.layout.max_width` is `100%` so the image fills the container. This will usually not preserve the aspect ratio of image.\n",
     "\n",
```
Comment (jasongrout) [inline docs/source/examples/Widget Styling.ipynb]: Can you delete this newline as well, so the cell starts with the next header?

**Comment by jasongrout** [comment]
Looks great, thanks! Can you make the one last change, and then I'll merge?

File: docs/source/examples/Widget Styling.ipynb (hunk 1 above)
Comment (jhl0204) [reply docs/source/examples/Widget Styling.ipynb]: Yep just deleted the newline at the end and pushed the commit.
Let me know if there is anything else / if I did something incorrectly!

**Comment by jasongrout** [comment]
Thanks!

**Pull request merged**